docs/source/config_file.rst

```diff
@@ -214,6 +214,35 @@ section of the command line docs.
 
     This option may only be set in the global section (``[mypy]``).
 
+    .. note::
+
+       Note that the TOML equivalent differs slightly. It can be either a single string
+       (including a multi-line string) -- which is treated as a single regular
+       expression -- or an array of such strings. The following TOML examples are
+       equivalent to the above INI example.
+
+       Array of strings:
+
+       .. code-block:: toml
+
+          [tool.mypy]
+          exclude = [
+              '^file1\.py$',  # literal (no escaping)
```

Comment:
```suggestion
              '^file1\.py$',  # a literal period should be escaped with a backslash
```

Reply:
> Are these comments cleared or not?

I _**think**_ the answer is no. (I don't know that anyone has reviewed them or who has authority to bless them.)

With regard to your proposed edits, I think they miss the mark. It sounds like you're reworking them to relate to regular expressions. The intention was to distinguish between [TOML's "literal" strings (which do not require or support escape sequences) and basic strings (which support escape sequences like `\\`, `\n`, etc.)](https://toml.io/en/v1.0.0#string). They are roughly analogous to Python's `r"…"` and `"…"`, respectively.

What do you think about this, instead?

``` toml
[tool.mypy]
exclude = [
    "^file1\\.py$",  # TOML's basic strings (double quotes) require escaping backslashes
    '^file2\.py$',  # but TOML's literal strings (single quotes) do not
```

Reply:
Your reformulation I makes the intent clear.  Let’s go with that.

Reply:
I tightened it up a bit and pushed a change with two slightly different versions. I'll @-mention you in those places to highlight them.